.github/workflows/scheduled.yml

```diff
@@ -34,6 +34,8 @@ jobs:
     runs-on: ubuntu-22.04
     needs: create_matrix
     name: "Online check: ${{ matrix.formula }}"
+    env:
+      HOMEBREW_GITHUB_API_TOKEN: "${{ github.token }}"
     strategy:
       matrix:
         formula: ${{ fromJson(needs.create_matrix.outputs.json) }}
```

Comment:
maybe good to also add `fail-fast: false`

Reply:
Yeah, I was thinking about it. Together with pre-installing homebrew curl in case it's needed. But that's all lower priority to me.